apps/showcase/project.json

```diff
@@ -96,6 +96,11 @@
             "glob": "*.json",
             "input": "packages/@o3r-training/training-sdk/dist/structure",
             "output": "/assets/@o3r-training/training-sdk/structure"
+          },
+          {
+            "glob": "*.json",
+            "input": "packages/@o3r-training/showcase-sdk/dist/structure",
```

Comment:
not part of this PR touched file but is this path part of the `^structure` inputName files?

Reply:
https://github.com/AmadeusITGroup/otter/blob/feat/otter-training/nx.json#L95 yes it is